### PR TITLE
Implemented new FTA tree dialog

### LINKED ISSUE #283: Implement new FTA tree dialog
It should be compliant with figma design. Please first investigate missing API and data on backend-side.

image

image

### PR DESCRIPTION
**Pull request by LaChope**
Resolves #283

### CODE REVIEW

File: public/locales/cs/translation.json [hunk 1]
```diff
@@ -19,7 +19,9 @@
     "title": "Vytvořit strom poruch",
     "namePlaceholder": "Název stromu poruch",
     "eventPlaceholder": "Událost",
-    "create": "Vytvořit novou událost",
+    "create": "Vytvořit strom poruch",
+    "noEvents": "No events found",
+    "createEvent": "Create New Event",
```
Comment (LaChope) [inline public/locales/cs/translation.json:24]: I could not find a good translation here.

Comment (blcham) [inline public/locales/cs/translation.json:24]:
```suggestion
    "noEvents": "Žádné události nenalezeny",
    "createEvent": "Vytvořit novou událost",
```

**Review by blcham — APPROVED** [review]
LGTM, can be merged

**Comment by blcham** [comment]
@shellyear have a peek to this PR.

**Pull request merged**